src/Pgsql/Insert.php

```
@@ -21,6 +21,31 @@ class Insert extends Common\Insert implements ReturningInterface
+
+    /**
+     *
+     * Adds or removes the `ON CONFLICT DO NOTHING` clause, the Postgres
+     * equivalent of the IGNORE flag on other databases.
+     *
+     * @param bool $enable Set or unset the clause (default true).
+     *
+     * @return $this
+     *
+     */
+    public function ignore($enable = true)
```

PHPStan (2.2.5)

[error] Return type ($this(Aura\SqlQuery\Pgsql\Insert)) of method Aura\SqlQuery\Pgsql\Insert::ignore() should be compatible with return type (Aura\SqlQuery\Sqlite\Insert) of method Aura\SqlQuery\Common\Insert::ignore()


(method.childReturnType)